src/Logger.hs

```diff
@@ -20,7 +20,5 @@ data LogEntry =
   | LogEntryForModerator EventLogItemCold
   deriving (Eq, Show)
 
-type SendLogMsg = LogEntry -> IO ()
```

Comment:
I would like to keep this type synonym. There is similar namings constructions in the persistent module.